api/router.go

```diff
@@ -75,6 +75,11 @@ func NewRouter(s *Server) *mux.Router {
 		"snapshotRevert": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotRevert),
 		"snapshotBackup": s.fwd.Handler(s.fwd.HandleProxyRequestByNodeID, s.fwd.GetHTTPAddressByNodeID(OwnerIDFromVolume(s.m)), s.SnapshotBackup),
 
+		"snapshotCRCreate": s.SnapshotCRCreate,
+		"snapshotCRList":   s.SnapshotCRList,
+		"snapshotCRGet":    s.SnapshotCRGet,
+		"snapshotCRDelete": s.SnapshotCRDelete,
```

Comment:
I think we cannot remove the old snapshot endpoints (actions) like snapshotGet, etc.. because of the reason explained here https://github.com/longhorn/longhorn-manager/pull/1541#issuecomment-1535494956

Reply:
As discussed in the meeting, I created a ticket to investigate this later https://github.com/longhorn/longhorn/issues/5894